recipes/adal/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "adal" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "120821f72ca9d59a7c7197fc14d0e27448ff8d331fae230f92d713b9b5c721f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pyjwt >=1.0.0
+    - requests >=2.0.0
+    - python-dateutil >=2.1.0
+    - cryptography >=1.1.0
+
+test:
+  imports:
+    - adal
+
+about:
+  home: https://github.com/AzureAD/azure-activedirectory-library-for-python
+  license: MIT
+  license_family: MIT
+  summary: The ADAL for Python library makes it easy for python application to authenticate to Azure Active Directory (AAD) in order to access AAD protected web resources.
```

Comment:
Please break this up over multiple lines

Reply:
I had that, but my syntax was invalid and then I couldn't find the spec fast enough, so I reverted it. I'll try it again